deps/rabbitmq_auth_backend_oauth2/README.md

```diff
@@ -139,6 +139,47 @@ In that case, the configuration will look like this:
 
 NOTE: `jwks_url` takes precedence over `signing_keys` if both are provided.
 
+### Variables Configurable in rabbitmq.conf
+
+| Key                                      | Documentation     
+|------------------------------------------|-----------
+| `auth_oauth2.resource_server_id`         | The Resource Server ID. Please see below for more details
+| `auth_oauth2.additional_scopes_key`      | Configure the plugin to also look in other fields (maps to `additional_rabbitmq_scopes` in the old format)
+| `auth_oauth2.default_key`                | ID of the default signing key
+| `auth_oauth2.signing_keys`               | Paths to signing key files
+| `auth_oauth2.jwks_url`                   | The URL of key server. According to the JWT Specification key server URL must be https.
+| `auth_oauth2.https.cacertfile`           | Path to a file containing PEM-encoded CA certificates. The CA certificates are used during key server authentication
+| `auth_oauth2.https.depth`                | Maximum number of non-self-issued intermediate certificates that can follow the peer certificate in a valid certification path. Default is 10. Please see: https://www.erlang.org/doc/man/ssl.html#type-allowed_cert_chain_length for more details
+| `auth_oauth2.https.peer_verification`    | Identify if the verification should be performed towards key server. Available values: `verify_none`, `verify_peer`. Default is `verify_none`. It is recommended to configure `verify_peer`
```

Comment:
I find this language very complex. "Enables or disables peer verification" would be simpler and would use the standard language we use everywhere in the TLS guide. It should be mentioned that peer verification requires a certain amount of setup and is more secure.